Rakefile

```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp1
```

Comment:
this is wrong. It is commit only for SLE12SP1 maintenance branch. Please revert this change.